operator/instance_group.go

```diff
@@ -51,6 +51,9 @@ func contains(instanceGroup set.Set, instanceTemplateName string) bool {
 // create instance group manager list
 func valuesIG(m map[string]compute.InstanceGroupManagersScopedList) []*compute.InstanceGroupManager {
 	var res []*compute.InstanceGroupManager
+	if m == nil {
+		return nil
+	}
```

Comment:
🔧I thing this needsto be changed.
This code should be unnecessary because this is illegal that map is nil.
Please investigate the cause of map becoming nil.